touchnet/models/qwen2_audio/__init__.py

```
@@ -0,0 +1,70 @@
+    # Reasoning behind the factor of 12 for the self-attention part of the formula:
+    # 1. each self-attention has 2 matmul in the forward and 4 in the backward (6)
+    # 2. the flash attention does 1 more matmul recomputation in the backward
+    #    but recomputation should not be counted in calculating MFU           (+0)
+    # 3. each matmul performs 1 multiplication and 1 addition                 (*2)
+    # 4. we follow the convention and do not account for sparsity in causal attention
+    flop_per_token = 6 * num_params + 12 * l * h * q * t
+
+    return flop_per_token
+
+
+def get_num_params(model: torch.nn.Module, exclude_embedding: bool = False) -> int:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)